installer_scripts/install-yb-voyager

```diff
@@ -262,9 +262,9 @@ install_debezium_server(){
 
 install_debezium_server_latest_release() {
 	output "Installing debezium:${DEBEZIUM_VERSION}"
-	debezium_server_filename="debezium-server-${DEBEZIUM_VERSION}.tar.gz"
+	debezium_server_filename="debezium-server.tar.gz"
```

Comment:
I think it would be good to keep the debezium version in the file name. 
shall we keep it for now? I realise the release automation will only create "debezium-server.tar.gz" but we should be able to rename the file manually. For the next release, we can automate the release automation as well to include the version.

Reply:
Will it be a lot of change in the pipelines for the other packaging tools?

Reply:
Hmm this will require me to change all the release pipelines too

Reply:
Can we make this into a ticket for the next release?

Reply:
Sure @makalaaneesh